tests/test_core.py

```diff
@@ -1708,6 +1709,8 @@ def test_runtime_stacksave(self):
     self.do_run(src, 'success')
 
   def test_memorygrowth(self):
+    if self.changed_setting('ALLOW_MEMORY_GROWTH'):
```

Comment:
Oh I see.   You're trying to exclude this test completely from certain test suites because the test itself makes change to that setting?    I guess that make sense.

Reply:
Yes, exactly - if a test changes a setting, and a test mode changes a setting, there is no meaningful way to run the test. So we skip those.